config.json

```diff
@@ -16,6 +16,16 @@
         "strings"
       ]
     },
+    {
```

Comment:
In this file, the core exercises come first. The side exercises, like this one, should go below the core exercises (Tournament is the last core.) Easiest is to move it to the end of the file. There's no real system other than having the cores first, but lately we started adding new ones to the end.  

You can of course wait to make the change until we decided on adding the exercise or not.

Reply:
I have updated the config.json file. Thanks!